src/components/FilterControls/MediaTitleSearchInput/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react'
+import FormControl from '@mui/material/FormControl'
+import TextField from '@mui/material/TextField'
+import useDebounce from '../../../hooks/useDebounce'
+
+interface MediaTitleSearchInputProps {
+  defaultValue?: string
```

Comment:
Hot take: I don't think you want to pass a default value here. In MUI at least, these values are usually reserved for something like a default numerical option in a field. Your fields don't have default options, and this is not something that ought to change over the lifetime of your app. I'd remove this prop. I'll repeat this elsewhere too.